bot/engine/src/main/kotlin/engine/config/ConfiguredStoryHandler.kt

```diff
@@ -292,7 +321,7 @@ internal class ConfiguredStoryHandler(
         return transformedAnswers
     }
 
-    private fun BotBus.send(container: StoryDefinitionAnswersContainer, answer: SimpleAnswer, end: Boolean = false) {
+    fun BotBus.send(container: StoryDefinitionAnswersContainer, answer: SimpleAnswer, end: Boolean = false) {
```

Comment:
```suggestion
    private fun BotBus.send(container: StoryDefinitionAnswersContainer, answer: SimpleAnswer, end: Boolean = false) {
```
Doit pouvoir rester private à priori même si je peux me tromper